tests/test_sanity.py

```diff
@@ -603,6 +606,7 @@ def test_emconfig(self):
       os.remove(custom_config_filename + "_sanity")
     shutil.rmtree(temp_dir)
 
+  @no_wasm_backend('depends on WASM=0 working')
```

Comment:
I think we can remove the WASM=0 instances - it would be faster, too. One guess is that they were put there when wasm used to trigger a binaryen build, but fixed that a while ago.